docs/system-admin-guide/users-permissions/users/README.md

```diff
@@ -7,150 +7,200 @@ robots: index, follow
+Also consider the **[authentication](#authentication) settings**. See [Manage user settings](#manage-user-settings) for full details.
 
-![add users to a group](image-20200115165406439.png)
+### Activate users
 
+Open the user list. If a user has created their own account it is shown in the list with an "Activate" link on the right. Click this link and continue to add details to this user as above. There is also an "Activate" button at the top of the user details page.
 
+## Authentication
 
-## Delete users
+The available authentication methods affect the content of the Authentication section in the user details.  See [authentication settings](../../authentication/authentication-settings/) for details.
 
-In order to delete users from the system, open the [user list](#user-list).
+Use the **self-registration** field to give the following controls over a new user's access.
 
-Click on the **user name** of the user which you want to delete.
+### Manual account activation
 
-To delete a user permanently from the system, press the **Delete** button at the top right.
+The user details Authentication section has fields **Assign random password**, **Password**, **Confirmation** and **Enforce password change**.
 
-![Sys-admin-delete-user](Sys-admin-delete-user.png)
+* If you are near the new user, you can enter a password and confirmation then tell the user what it is. They can then sign in. It is recommended that you also tick the enforce password change tickbox, so that the user is prompted to change their password after they sign in.
+* You can phone the new user or send them an email, not using OpenProject, to give them the password. In this case it is more important to tick the enforce password change tickbox.
+* Tick the Assign random password, and probably the enforce password change tickbox. When the details are saved OpenProject will send an email to the new user with their password.
 
-You will then be asked to type in the user's email address in order to delete the user permanently from the system.
+### Account activation by email
 
-![delete user](image-20200115162533470.png)
+Leave all fields blank. When the details are saved OpenProject will send an email to the new user with a link inviting the user to OpenProject. They click the link to get the registration page to complete creating their account.
 
-<div class="alert alert-info" role="alert">
-**Note**: Deleting a user account is an irreversible action and cannot easily be restored via the frontend.
-</div>
+### Resend user invitation via email
+
+If a user did not receive the email invitation, you can send the invitation to the user again if needed. In the user list, click on the user name to whom you want to resend the email with the invitation link to the system.
+
+In the top right, click the **Send invitation** button in order to send the email once again.
+
+![Sys-admin-resend-invitation](Sys-admin-resend-invitation.png)
+
+## FAQ - What do I configure before I add users?
```

Comment:
Great idea! 
I would suggest to move this whole section ("FAQ - What do I..." to another place. 
I moved it to https://github.com/opf/openproject/commit/d899e4e39a9b704ed1632c8d9b7622c4c0b5bb14 .
I hope that's okay for you?

Reply:
Accepted. I have removed it from my changes